pretix-setup/pretix.cfg

```diff
@@ -0,0 +1,30 @@
+[pretix]
+instance_name=cosmos_website_pretix
+url=http://localhost
+currency=EUR
+datadir=/var/pretix/data
+trust_x_forwarded_for=on
+trust_x_forwarded_proto=on
+
+[database]
+backend=mysql
+name=pretix
+user=cosmos_website_tester
+password=20201234
```

Comment:
I think it might be a good idea to create seperate users for this, and also restrict permission to only those necessary. I will also look into django vs django cms to make sure those are also separated if that is possible, as i remember there being sth about that in the django-cms docs.